client/packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -46,14 +53,22 @@ export const DataRow = <T extends RecordWithId>({
   const onRowClick = () => onClick && onClick(rowData);
   const paddingX = dense ? '12px' : '16px';
   const paddingY = dense ? '4px' : 0;
+  const Animation: FC<PropsWithChildren> = ({ children }) =>
+    isAnimated ? (
+      <Fade in={true} timeout={500}>
```

Comment:
The `Fade` animation is used to make the Expand look better - but does add ~176ms
I've made it conditional on a prop and enabled when expando is used